db/migrate/20230106012051_create_users.rb

```diff
@@ -0,0 +1,10 @@
+class CreateUsers < ActiveRecord::Migration[7.0]
+  def change
+    create_table :users do |t|
+      t.string :name
```

Comment:
```suggestion
      t.string :name, null: false
```

FYI
一般的にはこのようにバリデーションを付加しておきます！
tutorial はこのまま進めてもらって大丈夫です 👍

Reply:
なるほどです！ありがとうございます！